gameplay/gameengine/service.go

```diff
@@ -115,6 +120,31 @@ func contains(s []int, e int) bool {
 	return false
 }
 
+// Move get a player moves.
+func (service *Service) Move(ctx context.Context, matchID uuid.UUID, card Card) ([]int, error) {
```

Comment:
method move have to accept number of the square of the field where the player will go, and you must check it and update information in db